src/openrct2/paint/Paint.cpp

```diff
@@ -551,11 +549,10 @@ static void PaintAttachedPS(rct_drawpixelinfo* dpi, paint_struct* ps, uint32_t v
     {
         auto screenCoords = ScreenCoordsXY{ attached_ps->x + ps->x, attached_ps->y + ps->y };
 
-        auto imageId = PaintPSColourifyImage(
-            ImageId::FromUInt32(attached_ps->image_id, ps->tertiary_colour), ps->sprite_type, viewFlags);
```

Comment:
This is subtly different, as it used to set the tertiary colour from `ps` instead of `attached_ps`. Might not be a problem, but it’s something to watch.

Reply:
Can you think why it would be using the tertiary colour from the parent paint node, rather than the tertiary colour of the node that was added?

Reply:
We will still need to verify with some parks that there are no unexpected changes